src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -16,6 +16,13 @@
 from PyInstaller.utils.tests import importorskip, requires, xfail
 
 
+def test_fiona(test):
```

Comment:
The argument name matters here. It has to be `pyi_builder` to pick up the custom `pyi_builder` [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html). (It's not the most intuitive feature design.)